src/activities.ts

```diff
@@ -32,16 +28,13 @@ export async function deposit(details: PaymentDetails): Promise<string> {
   //   details.amount,
   //   details.referenceId
   // );
-  try {
-    const result = await bank2.deposit(
-      details.sourceAccount,
-      details.amount,
-      details.referenceId
-    );
-    return result;
-  } catch (error) {
-    throw new Error('Unexpected error occurred');
-  }
+  //
+  const result = await bank2.deposit(
+    details.targetAccount,
```

Comment:
this was changed from `sourceAccount` because the deposit is going to the target not the source